src/Sql/Sql.Test/ScenarioTests/DataClassificationTests.ps1

```diff
@@ -354,6 +354,120 @@ function Test-DataClassificationOnSqlDatabase
+		
+		# Get, using pipeline, recommended sensitivity labels, and verify.
+		$recommendations = Get-AzSqlDatabase -ResourceGroupName $params.rgname -ServerName $params.serverName -DatabaseName $params.databaseName | Get-AzSqlDatabaseSensitivityRecommendation
+		Assert-AreEqual 3 ($recommendations.SensitivityLabels).count
+		Assert-AreEqual $params.rgname $recommendations.ResourceGroupName
+		Assert-AreEqual $params.serverName $recommendations.ServerName
+		Assert-AreEqual $params.databaseName $recommendations.DatabaseName
+
+		# Verify disabled recommdation is not part of the new recommdations.
+		Assert-AreNotEqual $firstColumnName ($recommendations.SensitivityLabels)[0].ColumnName
+		Assert-AreNotEqual $firstColumnName ($recommendations.SensitivityLabels)[1].ColumnName
+		Assert-AreNotEqual $firstColumnName ($recommendations.SensitivityLabels)[2].ColumnName
+
+		# Disable, using pipeline, all recommended columns.
+		Get-AzSqlDatabase -ResourceGroupName $params.rgname -ServerName $params.serverName -DatabaseName $params.databaseName | Get-AzSqlDatabaseSensitivityRecommendation | Disable-AzSqlDatabaseSensitivityRecommendation
+		
+		# Verify no recommdations are retrieved since all are disabled.
+		$recommendations = Get-AzSqlDatabase -ResourceGroupName $params.rgname -ServerName $params.serverName -DatabaseName $params.databaseName | Get-AzSqlDatabaseSensitivityRecommendation
+		Assert-AreEqual $params.rgname $recommendations.ResourceGroupName
+		Assert-AreEqual $params.serverName $recommendations.ServerName
+		Assert-AreEqual $params.databaseName $recommendations.DatabaseName
+		Assert-AreEqual 0 ($recommendations.SensitivityLabels).count
+
+		# Enable, using pipeline, second disabled recommdation and verify
+		Get-AzSqlDatabase -ResourceGroupName $params.rgname -ServerName $params.serverName -DatabaseName $params.databaseName | Enable-AzSqlDatabaseSensitivityRecommendation -SchemaName $secondSchemaName -TableName $secondTableName -ColumnName $secondColumnName
+		
+		# Verify enabled recommdation is now part of the recommendations.
+		$recommendations = Get-AzSqlDatabase -ResourceGroupName $params.rgname -ServerName $params.serverName -DatabaseName $params.databaseName | Get-AzSqlDatabaseSensitivityRecommendation
+		Assert-AreEqual $params.rgname $recommendations.ResourceGroupName
+		Assert-AreEqual $params.serverName $recommendations.ServerName
+		Assert-AreEqual $params.databaseName $recommendations.DatabaseName
+		Assert-AreEqual 1 ($recommendations.SensitivityLabels).count
+
+		$recommendation = ($recommendations.SensitivityLabels)[0]
+		Assert-AreEqual $secondSchemaName $recommendation.SchemaName
+		Assert-AreEqual $secondTableName $recommendation.TableName
+		Assert-AreEqual $secondColumnName $recommendation.ColumnName
+		Assert-NotNullOrEmpty $recommendation.InformationType
+		Assert-NotNullOrEmpty $recommendation.SensitivityLabel
+	}
+	finally
+	{
+		# Cleanup
+		Remove-DataClassificationTestEnvironment $testSuffix
```

Comment:
Don't find the test cases for Enable/Disable-AzSqlInstanceDatabaseSensitivityRecommendation